src/test/java/org/example/carina/demo/api/RetrieveCollectionMethodTest.java

```diff
@@ -0,0 +1,32 @@
+package org.example.carina.demo.api;
+
+import com.qaprosoft.carina.core.foundation.IAbstractTest;
+import com.qaprosoft.carina.core.foundation.api.http.HttpResponseStatusType;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+
+import java.lang.invoke.MethodHandles;
+
+public class RetrieveCollectionMethodTest implements IAbstractTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    @Test()
+    @MethodOwner(owner = "pawelUlita")
```

Comment:
```suggestion
    @MethodOwner(owner = "pulita")
```
This is not necessary, but it is a company standard